src/chrome/content/rules/ArtStation.com.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="ArtStation">
+	<target host="artstation.com" />
+	<target host="*.artstation.com" />
```

Comment:
The majority of subdomains are user-created sites. The only official ones that I could find are `www`, `cdn`, `cdna`, and `cdnb`.